grid-opdracht.html

```diff
@@ -3,9 +3,14 @@
 <head>
     <meta charset="UTF-8">
     <link rel="stylesheet" href="styles.css">
-    <title>Les 3 - Opdracht CSS Grid</title>
+    <title>My New Grid 2</title>
 </head>
 <body>
-
+<aside class="container">
+    <header class="item" id="item-one">header</header>
+    <aside class="item" id="item-two">aside-left</aside>
+    <main class="item" id="item-three">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</main>
+    <aside class="item" id="item-four">aside-right</aside>
+    <footer class="item" id="item-five">footer</footer>
```

Comment:
Je mist hier een closing tag van regel 9. In jou geval `</aside>`, `</div>` was beter geweest.